packages/redux/src/promotionEvaluations/types/action.types.ts

```diff
@@ -0,0 +1,30 @@
+import type * as actionTypes from '../actionTypes';
+import type { Action } from 'redux';
+import type { Error } from '@farfetch/blackout-client/types';
+import type {
+  PromotionEvaluationId,
+  PromotionEvaluationItem,
+} from '@farfetch/blackout-client/promotionEvaluations/types';
+
+export interface FetchPromotionEvaluationItemsRequestAction extends Action {
+  meta: { promotionEvaluationId: PromotionEvaluationId };
+  type: typeof actionTypes.FETCH_PROMOTION_EVALUATION_ITEMS_REQUEST;
+}
+export interface FetchPromotionEvaluationItemsSuccessAction extends Action {
+  meta: { promotionEvaluationId: PromotionEvaluationId };
+  type: typeof actionTypes.FETCH_PROMOTION_EVALUATION_ITEMS_SUCCESS;
```

Comment:
Nitpick: it's nicer to have this alphabetically sorted 😇

Reply:
Done 🙂